README.md

```diff
@@ -1 +1,70 @@
-# math-magicians
\ No newline at end of file
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly update the README file with microverse [template](https://github.com/microverseinc/readme-template) instead of react template and customize it with your info.